harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/csw/Aligner.java

```diff
@@ -474,8 +472,8 @@ boolean updatingLocalMetadata(RecordInfo ri, String id, Boolean force) throws Ex
         String language = context.getLanguage();
         final AbstractMetadata metadata = metadataManager.updateMetadata(context, id, md, validate, ufo, language, ri.changeDate, true, IndexingMode.none);
 
-        if (force || updateSchema) {
-            if (force) {
+        if (Boolean.TRUE.equals(force) || updateSchema) {
```

Comment:
Same here, if you change the `Boolean force` to `boolean force`, you can still use the same simple logic without any warning and avoid boxing/unboxing the value

Reply:
Same comment as previous.

Reply:
Checked the code and it was safe to use the primitive `boolean`. Code updated